2.x: Update about usage of branches in Level1 automation

warpech opened the pull request:
Summary in the commit message

warpech committed:
say that `2.x` is the not used anymore in Level1 automation of Starcounter 2.4

alshakero commented on CONTRIBUTING.md:
```diff
@@ -1,8 +1,8 @@
 ## Usage of StarcounterClientFiles 2.x in Level1 build automation
 
-Whatever is pushed on branch `2.x`, will be bundled in the Starcounter installer by the Level1 build automation. See [the announcement](https://github.com/Starcounter/AdminTrack/issues/438) for more details. 
+Whatever is pushed on branch `2.x`, in NOT bundled in the Starcounter installer by the Level1 build automation.
```
I assume you mean "**is** not".

Anyways I don't see the need for "whenever" any more. "Whenever X happens Y won't happen". Doesn't really follow. 

I would go for "Branch `2.x` is NOT bundled in the Starcounter installer by the Level1 build automation."

warpech committed:
clarify the branch explanation

as proposed in https://github.com/Starcounter/StarcounterClientFiles/pull/46#discussion_r170124859

warpech commented:
Thanks for the review!
